packml_devel/src/idleState.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+
+import roslib; roslib.load_manifest('test')
```

Comment:
This should no longer be needed.  An OK example of a proper python ROS file can be found [here](https://github.com/ros-industrial/industrial_core/blob/kinetic-devel/industrial_robot_simulator/industrial_robot_simulator).